debian/changelog

```diff
@@ -1,3 +1,53 @@
+ubuntu-advantage-tools (27.13~23.04.1) lunar; urgency=medium
+
+  * d/bash-completion:
+    - enable autocomplete for the 'pro' command (GH: #2280)
+  * d/control:
+    - update the package description
+  * d/postinst:
+    - remove unauthenticated esm repos from Xenial systems (LP: #1990378)
+  * New upstream release 27.13 (LP: #2003018)
+    - apt:
+      + remove logic which added repositories and pinned them to 'never' to
+        enable access to esm package lists
+      + add functionality to create and update a local apt esm cache with
+        the lists for esm-infra and esm-apps
+    - apt-hook: update the cpp hook to use the local esm apt cache
+    - apt-news:
+      + fetch and display APT News in apt upgrade
+      + show contract expiration notices in the apt news output
+    - attach: support attaching without being able to install snapd
+      (LP: #1997514)
+    - cli:
+      + do not show invalid subcommands in autocomplete (GH: #2279)
+      + add commands to support the magic-attach flow
```

Comment:
It is not a new command per se, since it is part of `pro attach`. I think we can change it to `add support for attaching without a token`